<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="genshinImg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>res/ico/genshinico.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="starrailImg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>res/ico/starrailico.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="zzzImg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>res/ico/zzzico.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="genshinTimer">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignCenter|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="starrailTimer">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignCenter|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="zzzTimer">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignCenter|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="genshinInput">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="genshinButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>更新</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="genshin_notify">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>95</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property>
    </property>
     <string>全快する1時間前に通知する</string>
    </property>
   </widget> 
   <widget class="QLineEdit" name="starrailInput">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zzzInput">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="starrailButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>更新</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="starrail_notify">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property>
    </property>
     <string>全快する1時間前に通知する</string>
    </property>
   </widget> 
   <widget class="QPushButton" name="zzzButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>更新</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="zzz_notify">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property>
    </property>
     <string>全快する1時間前に通知する</string>
    </property>
   </widget> 
   <widget class="QFrame" name="gensin_rpb_container">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="RoundProgressBar" name="genshin_rpb" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>130</width>
       <height>130</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="starrail_rpb_container">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="RoundProgressBar" name="starrail_rpb" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>130</width>
       <height>130</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="zzz_rpb_container">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="RoundProgressBar" name="zzz_rpb" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>130</width>
       <height>130</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
    </widget>
   </widget>
 <customwidgets>
  <customwidget>
   <class>RoundProgressBar</class>
   <extends>QWidget</extends>
   <header>PySideExtn.RoundProgressBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
